src/Quasar/Mount/Common/Gen.purs

```diff
@@ -67,17 +66,10 @@ genCredentials =
     <*> Gen.choose (pure "") genAlphaNumericString)
 
 genDirPath ∷ ∀ m. MonadGen m ⇒ MonadRec m ⇒ m DirPath
-genDirPath = Gen.sized \size → do
-  newSize ← Gen.chooseInt 0 size
-  Gen.resize (const newSize) do
-    parts ∷ L.List String ← Gen.unfoldable genAlphaNumericString
-    pure $ foldr (flip P.appendPath <<< P.dir) P.rootDir parts
+genDirPath = PGen.genAbsDirPath
 
 genFilePath ∷ ∀ m. MonadGen m ⇒ MonadRec m ⇒ m FilePath
-genFilePath = do
-  dir ← genDirPath
-  file ← genAlphaNumericString
-  pure $ dir </> P.file file
+genFilePath = PGen.genAbsFilePath
```

Comment:
Why do we need this be redefined here? You can just reexport them and rename, right? Anyways that's not public API

Reply:
kk will rename